features/scroll-buttons.yml

```diff
@@ -0,0 +1,9 @@
+name: ::scroll-button
+description: The `::scroll-button(<dir>)` CSS pseudo-element is a focusable button which scrolls its originating scroll container element in the specified direction.
+spec: https://drafts.csswg.org/css-overflow-5/#scroll-buttons
+group: scrolling
+caniuse: css-scroll-buttons
```

Comment:
Let's remove this line, there's no caniuse feature called css-scroll-buttons yet. When there is, we can add it.
See also: https://github.com/web-platform-dx/web-features/blob/main/docs/CONTRIBUTING.md, which has some information about caniuse.

Reply:
Sounds good, I've removed the caniuse reference in all the PRs